lib/web/desktop.go

```diff
@@ -118,11 +118,11 @@ func (h *Handler) createDesktopConnection(
 	if username == "" {
 		return sendTDPError(ws, trace.BadParameter("missing username"))
 	}
-	width, err := strconv.Atoi(q.Get("width"))
+	width, err := apiutils.StrToUInt32(q.Get("width"))
```

Comment:
I don't think this is necessary here. There's already a check for max width height on line 130 (which is a stricter check than max uint32 anyway).